CHANGELOG.md

```diff
@@ -9,12 +9,16 @@ IMPROVEMENTS:
 
 * Data Source: `azurerm_databricks_workspace` - support for the `tags` property [GH-9933]
 * `azurerm_api_management` - support `Consumption_0` for the `sku_name` property [GH-6868]
+* `azurerm_app_service` - support `service_tag` in `ip_restriction` [GH-9609]
+* `azurerm_app_service_slot` - support `service_tag` in `ip_restriction` [GH-9609]
 * `azurerm_cdn_endpoint` - only send `content_types_to_compress` and `geo_filter` to the API when actually set [GH-9902]
 * `azurerm_cosmosdb_mongo_collection` - correctly read back the `_id` index when mongo 3.6 [GH-8690]
 * `azurerm_container_group` - support for the `volume.empty_dir` property [GH-9836]
 * `azurerm_data_factory_linked_service_azure_file_storage` - support for the `file_share` property [GH-9934]
 * `azurerm_dedicated_host` - support for addtional `sku_name` values [GH-9951]
 * `azurerm_function_app` - make `pre_warmed_instance_count` computed to use azure's default [GH-9069]
+* `azurerm_function_app` - support `service_tag` in `ip_restriction` [GH-9609]
+* `azurerm_function_app_slot` - support `service_tag` in `ip_restriction` [GH-9609]
```

Comment:
Could you revert changes to CHANGELOG.md please? This is updated by the maintainers via another process when a PR is accepted, tested and merged.

Reply:
Done, I just rebased away the commit. I'd love to see a `CONTRIBUTING` file that clearly communicated the expectations for Pull Requests to be expected.

Reply:
We're working on that, hopefully not too far out after we catch up from the holidays 👍